README.md

```diff
@@ -19,6 +19,8 @@ _NOTE: You can change the port via the `GRAPHED_RESULTS_PORT` environment variab
 
 ## How to integrate it with Ringtail
 1. Log in to Ringtail.
+1. Verify an Organization has been created and appropriate cases and users have been assigned to, including the user that is performing ther rest of this procedure.
+1. On the `Portal Management -> User administration` page selet the user that is performing this procedure.  On the left menu select `API Access` and check the box to authorize access to the API.  Repeat this step for all users that will use the extension.
```

Comment:
You don't need to grant API access to users for UI Extensions to work with the API. UIX get access regardless. You only need to grant users access for them to use the Connect API explorer or to make API calls externally with their credentials.

Reply:
Interesting.  If you don't have API access the UI Extensions UI behaves badly when creating new extensions.  The user could add the extension but when you SAVE it, nothing happened (apparently).  After we granted API access and went back to extensions, there were a dozen entries from we clicked Save and nothing appeared to happen.... it did, we just didn't see it.

Reply:
Ah! That sounds like a bug we have in our backlog. I'll add this repro to it.

Thanks for reporting it!